scripts/deploy.sh

```diff
@@ -472,6 +474,8 @@ _deploy() {
   _subst_config "$scripts_path/$TELEMETRYUI_INI" "$telemetryui_path/"
   _subst_config "$scripts_path/uwsgi.conf"
   _subst_config "$scripts_path/sites_nginx.conf"
+  # get uwsgi location, in case this was installed during script exec
+  UWSGI_PATH=$(which uwsgi)
```

Comment:
I would prefer to use the `type` bash builtin, rather than `which`, since it may or may not be installed.